Remove client.getStreamPartsByStorageNode() and update related cli-tools command (#485)
streamr-dev/network: packages/cli-tools/CHANGELOG.md

## [Unreleased]

### Changed

> - Rename command `storage-node list-stream-parts` to `storage-node list-streams` and change the output format

Reviewer: Just thought about this, it's more like the command "replaced" the old one than "renamed" since "rename" refers to the functionality staying the same (it's not since it now lists stream instead of stream parts).

```suggestion
- Replace command `storage-node list-stream-parts` with `storage-node list-streams` and change the output format
```
